build.gradle

```diff
@@ -221,6 +231,20 @@ subprojects {
 		}
 	}
 
+	jar {
+		manifest {
+			attributes(
+				'Built-By': System.properties['user.name'],
+				'Created-By': "${System.properties['java.version']} (${System.properties['java.vendor']} ${System.properties['java.vm.version']})".toString(),
+				'Build-Date': buildDate,
+				'Build-Time': buildTime,
+				'Build-Revision': versioning.info.commit,
+				'Specification-Title': project.name,
+				'Specification-Version': project.version,
```

Comment:
AFAIK Gradle has a built-in version parsing utility that can be used to correctly detect the version number.